depthai_descriptions/launch/urdf_launch.py

```diff
@@ -48,9 +49,32 @@ def launch_setup(context, *args, **kwargs):
                     'cam_pitch:=', cam_pitch, ' ',
                     'cam_yaw:=', cam_yaw
                 ])}]
-        )
+            ),
+            LoadComposableNodes(
+            target_container=name+"_container",
+            condition=IfCondition(use_composition),
+            composable_node_descriptions=[
+                ComposableNode(
+                    package='robot_state_publisher',
+                    plugin='robot_state_publisher::RobotStatePublisher',
+                    name=name+'_state_publisher',
+                    parameters=[{'robot_description': Command(
+                        [
+                            'xacro', ' ', xacro_path, ' ',
+                            'camera_name:=', tf_prefix, ' ',
+                            'camera_model:=', camera_model, ' ',
+                            'base_frame:=', base_frame, ' ',
+                            'parent_frame:=', parent_frame, ' ',
+                            'cam_pos_x:=', cam_pos_x, ' ',
+                            'cam_pos_y:=', cam_pos_y, ' ',
+                            'cam_pos_z:=', cam_pos_z, ' ',
+                            'cam_roll:=', cam_roll, ' ',
+                            'cam_pitch:=', cam_pitch, ' ',
+                            'cam_yaw:=', cam_yaw
+                        ])}]
+                )])
+            ]
```

Comment:
isn't this set from the TFPublisher ?

Reply:
I wanted to keep old behavior for now so that users won't get unexpected changes.

Reply:
Lets add some warning. it will be deprecated ? They can still send these arguments. but those can be moved into TFPublisher config and do the config from there since that would be needed when mounting it on robot anyways. 

Only the camera model would be infered from the device unless specified. Thoughts?

Reply:
Since TFPublisher overrides this configuration it won't matter much what's set here if user enables it, though I could add another argument that launches the description without sensor extrinsics
- Arguments will be passed to the parameters in `camera.launch.py` and other launch files as well
- TF Publishing/overriding will be disabled by default (for now, I'll change it when default IMU extrinsics will be available from the firmware). I'll probably add another argument to change that
- Camera model, if not specified is taken from the device, and if there's no STL available for that particular one it uses OAK-D by default